demisto_sdk/commands/init/initiator.py

```diff
@@ -192,15 +194,22 @@ def convert_contribution_to_pack(self):
                     metadata = json.loads(metadata_file.read())
                     # a name passed on the cmd line should take precedence over one pulled
                     # from contribution metadata
-                    pack_name = self.name or self.format_pack_dir_name(metadata.get('name', 'ContributionPack'))
+                    pack_display_name = self.name or metadata.get('name', 'ContributionPack')
+                    # Strip 'Pack' suffix from pack display name if present
+                    pack_display_name = pack_display_name.strip()
+                    if pack_display_name.casefold().endswith('pack') > len(pack_display_name) > 4:
+                        stripped_pack_display_name = pack_display_name[:-4].strip()
+                        pack_display_name = stripped_pack_display_name or pack_display_name
+                    pack_name = self.dir_name or self.format_pack_dir_name(
+                        metadata.get('name', 'ContributionPack')
+                    )
                     # a description passed on the cmd line should take precedence over one pulled
                     # from contribution metadata
                     metadata_dict['description'] = self.description or metadata.get('description')
-                    metadata_dict['name'] = pack_name
-                    metadata_dict['author'] = metadata.get('author', '')
-                    metadata_dict['support'] = metadata.get('support', '')
+                    metadata_dict['name'] = pack_display_name
+                    metadata_dict['author'] = self.author or metadata.get('author', '')
+                    metadata_dict['support'] = 'community'
                     metadata_dict['url'] = metadata.get('supportDetails', {}).get('url', '')
-                    metadata_dict['email'] = metadata.get('supportDetails', {}).get('email', '')
```

Comment:
lets not remove the `email` field completely, just initialize it with empty string

Reply:
The `email` field is initialized with the empty string later in the `create_metadata` method.